src/Classes/ServiceAPI/MyRadio_List.php

```diff
@@ -345,6 +368,27 @@ public static function getAllLists()
         return $lists;
     }
 
+    /**
+     * Return mailing lists a user has access to.
+     *
+     * @return MyRadio_List[]
+     */
+    public static function getUserLists($memberid = -1)
+    {
+        $member = MyRadio_User::getInstance($memberid)->getID();
```

Comment:
what happens when memberid=-1 is passed in, as per default (why is there a default?)